mibs/BWS-MIB

```diff
@@ -0,0 +1,3211 @@
+BWS-MIB DEFINITIONS ::= BEGIN
+
```

Comment:
This file should be migrated to mibs/barracuda folder.

Reply:
Sorry, will do